src/frontend/screens/Game/GameSubMenu/index.tsx

```diff
@@ -233,6 +233,24 @@ export default function GamesSubmenu({
     isInstalled &&
     !isThirdPartyManaged
 
+  const hasWine = !is.win && !is.native
```

Comment:
We also need to check if that `gameSettings.wineVersion.type !== "crossover"` cause in that case it uses a crossover bottle and not a prefix

same check is done on the Installed Information paths https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/3f7033e7b37f4ccddec73f5187a98c9956909920/src/frontend/screens/Game/GamePage/components/InstalledInfo.tsx#L113

Reply:
In that case would it be appropriate to rename the option to `Browse CrossOver Bottle` when we're dealing with a game that uses CrossOver? Referencing this string:
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/3f7033e7b37f4ccddec73f5187a98c9956909920/public/locales/en/gamepage.json#L210

Reply:
I don't think users need that, bottles should be handle with Crossover's UI so it's better to not let the user mess with the bottle directly

Reply:
I've added a check for the wine type so that it doesn't show the "Browse Wine Prefix" option when the game uses Crossover.